modules/lib.lua

```diff
@@ -147,6 +148,20 @@ function qbx.table.mapBySubfield(tble, subfield)
     return map
 end
 
+---Returns true if the given array contains the given value.
+---@param arr any[]
+---@param val any
```

Comment:
Maybe consider adding a generic here instead of `any`.

Reply:
Good call, done.